.github/workflows/docker-build-push-ecr.yaml

```diff
@@ -58,15 +64,19 @@ jobs:
       - name: Check out code
         uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4
 
-      - name: Download artifacts to pass to docker build
-        if: ${{ inputs.artifact_name || inputs.artifact_path }} # avoid downloading artifacts if not needed
-        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4
+      - name: Download Artifacts
         env:
-          ARTIFACT_NAME: ${{ inputs.artifact_name || vars.artifact_name }}
-          ARTIFACT_PATH: ${{ inputs.artifact_path || vars.artifact_path }}
+          ARTIFACT_PATH: ${{ inputs.artifact_path || vars.artifact_path || '' }}
```

Comment:
You can remove the the | '' since thats the default behaviour of an unset input or var

Reply:
ok